### DISCUSSION TITLE
Suricata alerts missing after reboot

### DISCUSSION THREAD

**Discussion by giomke** [Security-Onion-Solutions/securityonion#12371] [Unsupported Versions]
I noticed when I'm rebooting the server suricata alerts disappearing and start collecting new ones and I can't find previous alerts.
Is it normal? How can I avoid this happening ?

**Comment by cm-ops** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 can you give some more insight on your grid please?
